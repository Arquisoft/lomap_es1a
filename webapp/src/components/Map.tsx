import { Link } from "react-router-dom";

import Drawer from "@mui/material/AppBar";
import ShoppingCartIcon from "@mui/icons-material/ShoppingCart";
import AppBar from "@mui/material/AppBar";
import Badge from "@mui/material/Badge";
import Box from "@mui/material/Box";
import IconButton from "@mui/material/IconButton";
import Stack from "@mui/material/Stack";
import { styled } from "@mui/material/styles";
import placeholder_map from '../images/placeholder_map.png';

type MapProps = {
    // TODO: None yet
}

export default function Map(props: MapProps): JSX.Element {
    return (
        <img width="600" height="500" src={placeholder_map} alt="Map" />
    );
};