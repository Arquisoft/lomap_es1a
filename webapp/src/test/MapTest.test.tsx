import React from 'react';
import { render, screen } from '@testing-library/react';
import Map from '../components/Map';

describe('Map', () => {
  test('renders with correct props', () => {
    const lng = 2.1234;
    const lat = 48.5678;
    const zoom = 10;
    const mapWidth = '100%';
    const mapHeight = '400px';
    const mapTheme = 'light';
    render(
      <Map
        lng={lng}
        lat={lat}
        zoom={zoom}
        mapWidth={mapWidth}
        mapHeight={mapHeight}
        mapTheme={mapTheme}
      />
    );
    expect(screen.getByTestId('map')).toBeInTheDocument();
  });

  

});