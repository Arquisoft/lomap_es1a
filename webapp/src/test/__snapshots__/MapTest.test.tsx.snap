// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RenderMap 1`] = `ShallowWrapper {}`;
